README.md

```diff
@@ -28,7 +28,7 @@ The easiest way to get a transcript for a given video is to execute:
 ```python
 from youtube_transcript_api import YouTubeTranscriptApi
 
-YouTubeTranscriptApi.get_transcript(video_id)
+YouTubeTranscriptApi.get_transcript("video_id")

```

Comment:
Actually, this wasn't really done on accident. I just assumed that it would be obvious that a variable named `video_id` containing the video's id has to be defined in this scope. Do you feel that this is unclear? If so, I would suggest initialising `video_id = '<video_id>'` after the import and keep using `video_id`.